iina/VideoView.swift

```diff
@@ -454,10 +452,10 @@ extension VideoView {
 }
 
 fileprivate func displayLinkCallback(
-  _ displayLink: CVDisplayLink, _ inNow: UnsafePointer<CVTimeStamp>,
-  _ inOutputTime: UnsafePointer<CVTimeStamp>,
-  _ flagsIn: CVOptionFlags,
-  _ flagsOut: UnsafeMutablePointer<CVOptionFlags>,
+  _: CVDisplayLink, _: UnsafePointer<CVTimeStamp>,
+  _: UnsafePointer<CVTimeStamp>,
+  _: CVOptionFlags,
+  _: UnsafeMutablePointer<CVOptionFlags>,
   _ context: UnsafeMutableRawPointer?) -> CVReturn {
```

Comment:
Although unused with respects to the code, these parameter names effectively provide documentation for developers reading the code. See what the senior developers say, but I am against removing parameter names.

Reply:
Hmm, I tend to disagree here. My two points:
1. In my opinion, it actually better to omit some stuff that you don't need to perform your logic, you have less things to grasp and see only the important ones
2. If you want to change/debug this piece of code, it's better if you're familiar with it a bit more than just knowing the parameter names. It can be done really easily by checking where this function is used (`CVDisplayLinkSetOutputCallback`) and locating the parameter (`CVDisplayLinkOutputCallback`). After that you'll see the complete documentation with everything you may need

But, of course, I understand that it can be a hot topic, so I'm fully okay to return parameter names in every function even if it's not used

Reply:
On this one lets see what the senior developers think.

Reply:
I'd like to keep it as it is.